website/docs/r/kubernetes_cluster.html.markdown

```diff
@@ -202,7 +204,7 @@ resource "azurerm_kubernetes_cluster" "example" {
 
 ```
 
-* `role_based_access_control` - (Optional) A `role_based_access_control` block. Changing this forces a new resource to be created.
+* `role_based_access_control_enabled` (Optional) - Whether Role Based Access Control for the Kubernetes Cluster should be enabled. Defaults to `true`. Changing this forces a new resource to be created.
```

Comment:
this'd be a breaking change right now, should this be feature-flagged behind 3.0?